src/components/modals/ManageAntModal/ManageAntModal.tsx

```diff
@@ -0,0 +1,277 @@
+        </td>,
+        <button className="button" key={`${contractId}-controller-edit-button`}>
+          <PencilIcon width={20} height={20} fill="var(--text-white)" />
+        </button>,
+      ],
+      [
+        <td className="assets-table-item" key={`${contractId}-data`}>
+          Undernames:
+        </td>,
+        <td
+          className="assets-table-item"
+          style={{ flex: 4 }}
+          key={`${contractId}-data`}
+        >
+          {state?.records
+            ? `${Object.keys(state.records).length - 1} / 100`
+            : 'N/A'}
+        </td>,
+        <button className="button" key={`${contractId}-records-button`}>
+          <PencilIcon width={20} height={20} fill="var(--text-white)" />
+        </button>,
+      ],
+      [
+        <td className="assets-table-item" key={`${contractId}-data`}>
+          Owner:
+        </td>,
+        <td
+          className="assets-table-item"
+          style={{ flex: 4 }}
+          key={`${contractId}-data`}
+        >
+          {state?.owner ? state.owner.toString() : 'N/A'}
+        </td>,
+        <button
+          className="assets-manage-button"
+          key={`${contractId}-transfer-button`}
+        >
+          Transfer
+        </button>,
+      ],
+    ];
+    setAntDetails(details);
+  }
+
+  return (
+    // eslint-disable-next-line
+    <div
+      className="modal-container"
+      style={{ background: '#1E1E1E' }}
+      ref={modalRef}
+      onClick={handleClickOutside}
+    >
+      <div className="flex-column" style={{ margin: '10%' }}>
+        <div
+          className="flex"
+          style={{
+            justifyContent: 'flex-start',
+            width: '100%',
+          }}
+        >
+          <span className="flex bold text-medium white">
+            <NotebookIcon width={25} height={25} fill={'var(--text-white)'} />
+            &nbsp;Manage ANT:&nbsp;
+            <span className="flex">
+              {isMobile ? (
+                <CopyTextButton
+                  displayText={`${contractId
+                    .toString()
+                    .slice(0, 10)}...${contractId.toString().slice(-10)}`}
+                  copyText={contractId.toString()}
+                  size={24}
+                  wrapperStyle={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    textColor: 'var(--bright-white)',
+                  }}
+                />
+              ) : (
+                <CopyTextButton
+                  displayText={contractId.toString()}
+                  copyText={contractId.toString()}
+                  size={24}
+                  wrapperStyle={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    textColor: 'var(--bright-white)',
+                  }}
+                />
+              )}
```

Comment:
it's more concise to do this in the copy text component
```suggestion
              {isMobile ? (
                <CopyTextButton
                  displayText={isMobile ? `${contractId
                    .toString()
                    .slice(0, 10)}...${contractId.toString().slice(isMobile ? -10): contractId.toString()}`}
                  copyText={contractId.toString()}
                  size={24}
                  wrapperStyle={{
                    display: 'flex',
                    justifyContent: 'center',
                    alignItems: 'center',
                    textColor: 'var(--bright-white)',
                  }}
                />
              ) : (
                <CopyTextButton
                  displayText={contractId.toString()}
                  copyText={contractId.toString()}
                  size={24}
                  wrapperStyle={{
                    display: 'flex',
                    justifyContent: 'center',
                    alignItems: 'center',
                    textColor: 'var(--bright-white)',
                  }}
                />
              )}
```

Reply:
added